environmentManager/index.js

```diff
@@ -0,0 +1,73 @@
+const settings = require('../utils/parserConfigs')();
+const { validEnvironment } = require('../utils/parserConfigs/validators');
+const { getSplitFactory } = require('../sdk');
+
+class EnvironmentManager {
+  
+  constructor() {
+    
+    // In environments are the apiKey, factory an clientReadiness status related to an authToken
```

Comment:
Is this sentence okey? `In environments are ...` or `Environments are ...` ?

Reply:
I mean environments as the array of objects, not as what environments means to us. trying to say that inside environments array we can find objects with apiKey, factory and clientReadiness